app/assets/javascripts/annotations.js

```diff
@@ -19,12 +19,13 @@ $("#highlightLongLines").click(function() {
 });
 
 var initializeAnnotationsForCode = function() {
-  
+  window.annotationMode = "Code";
```

Comment:
It'd probably be good to have a master list/enum of all the constants that `annotationMode` can take on declared at the top of the file.